app/mailers/message_mailer.rb

```diff
@@ -1,39 +1,21 @@
 class MessageMailer < ActionMailer::Base
   include SummaryHelper
-  include NicknameHelper
   include MarkdownHelper
   include ConversationHelper
 
-  def created(message_id, recipient_id)
-    @message = Message.find(message_id)
-    @recipient = Person.find(recipient_id)
-    @message_markdown = markdown(@message.content)
-    @message_text = stripdown(@message.content)
-
-    @conversation = @message.conversation
-    @account = @conversation.account
-
-    subject = summary(@conversation)
```

Comment:
I'm not sure why this was added but let's keep it until we know we can remove it. The markdown stuff above, I'm pretty sure we'll need.